mkdocs.yml

```diff
@@ -84,7 +84,9 @@ pages:
     - 'Resize': resize.md
     - 'RGB to HSV': rgb2hsv.md
     - 'RGB to LAB': rgb2lab.md
-    - 'Rotate Image': rotate.md
+    - 'Rotate Tools':
```

Comment:
I propose we switch the order of the items to put the new function above the one we are deprecating. I also propose a slight name change since "New" tends to get dated pretty fast. How about "Rotate" and "Rotate Image" to match the names of the functions.